build_tools/circle/build_test_arm.sh

```diff
@@ -21,39 +21,53 @@ source build_tools/shared.sh
 
 sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
 sudo apt-get update
-sudo apt-get install python3-virtualenv ccache
-python3 -m virtualenv --system-site-packages --python=python3 testenv
-source testenv/bin/activate
-pip install --upgrade pip
+
+# Setup conda environment
+MINICONDA_URL="https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh"
+
+# Install Mambaforge
+wget $MINICONDA_URL -O mambaforge.sh
+MINICONDA_PATH=$HOME/miniconda
+chmod +x mambaforge.sh && ./mambaforge.sh -b -p $MINICONDA_PATH
+export PATH=$MINICONDA_PATH/bin:$PATH
+mamba update --yes conda
+
+# Create environment and install dependencies
+mamba create -n testenv --yes python=3.7
+source activate testenv
+
+# Use the latest by default
+mamba install --verbose -y  ccache \
+                            pip \
+                            numpy \
+                            scipy \
+                            cython \
```

Comment:
You can add `$(get_dep lib $LIB_VERSION)` for the others and set them to "latest" in the circle ci config file for consistency with other CI tools.

Reply:
Currently in `get_dep`, if a `*_VERSION` is not set, nothing is return and the associated package won't be installed.

Should we add this case to this branch here?
https://github.com/scikit-learn/scikit-learn/blob/1ea9ae2245046a6194dfc3f7125c1140804e18a2/build_tools/shared.sh#L10-L12

Or ask for the definition of a environment variable in the case of it not being set?

Reply:
> Currently in get_dep, if a *_VERSION is not set, nothing is return and the associated package won't be installed.

Which is a feature, not a bug: it makes it possible to have CI entries with and without the soft dependencies (pandas, matplotlib...). Please just add the needed environment variables.